device/src/graphics/widgets/backup_result_screen.rs

```diff
@@ -0,0 +1,115 @@
+use embedded_graphics::pixelcolor::Rgb565;
+use embedded_graphics::prelude::*;
+use embedded_graphics::prelude::*;
+use embedded_graphics::primitives::{PrimitiveStyle, Rectangle};
+use embedded_graphics::text::Text;
+use embedded_iconoir::{icons, Icon};
+use u8g2_fonts::U8g2TextStyle; // Assuming these exist in embedded_iconoir
+
+pub struct BackupResultScreen {
+    area: Size,
+    ok: bool,
+    // Only used for invalid backup (to offer a retry)
+    try_again_rect: Rectangle,
+}
+
+impl BackupResultScreen {
```

Comment:
Is this widget for a later PR? It isn't integrated yet

Reply:
Good catch. No this was some vibe coded thing I did at some point that got committed. I'll remove it.